sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRepositoryAsyncClientUpdateIntegrationTests.java

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.containers.containerregistry;
+
+import com.azure.containers.containerregistry.models.RegistryArtifactProperties;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.test.TestMode;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.azure.containers.containerregistry.TestUtils.DIGEST_UNKNOWN;
+import static com.azure.containers.containerregistry.TestUtils.DISPLAY_NAME_WITH_ARGUMENTS;
+import static com.azure.containers.containerregistry.TestUtils.HELLO_WORLD_SEATTLE_REPOSITORY_NAME;
+import static com.azure.containers.containerregistry.TestUtils.TAG_TO_DELETE;
+import static com.azure.containers.containerregistry.TestUtils.TAG_TO_UPDATE;
+import static com.azure.containers.containerregistry.TestUtils.TAG_UNKNOWN;
+
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
Is this being done due to tests sharing state? Could the test cases be update to make them more idempotent?

Reply:
Unfortunately they can't. Mentioned in the other thread.